ansible/.env.j2

```diff
@@ -0,0 +1,16 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=secret-key
+
+TRELLO_API_KEY={{trello_api_key}}
+TRELLO_TOKEN={{trello_token}}
+TRELLO_BOARD_ID={{trello_board_id}}
+TO_DO_LIST_NAME=To Do
```

Comment:
Could make these variables too but left as constants mainly out of laziness when being prompted for the variables...